.github/workflows/build-published.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches:
+      - published
+      # TODO remove the following line before merging this PR
```

Comment:
Looks like this needs to be removed before merging?

Reply:
Correct, this was to run the workflow once in this PR to test it. I've updated the PR.